packages/docbook_xsl.rb

```diff
@@ -25,10 +25,9 @@ class Docbook_xsl < Package
 
   depends_on 'docbook_xml'
   depends_on 'xmlcatmgr'
-  depends_on 'bash'
 
   def self.patch
-    system 'curl -OLf "https://github.com/archlinux/svntogit-packages/raw/packages/docbook-xsl/trunk/765567_non-recursive_string_subst.patch"'
+    system 'curl -OLsf "https://github.com/archlinux/svntogit-packages/raw/packages/docbook-xsl/trunk/765567_non-recursive_string_subst.patch"'
```

Comment:
Would it make sense to exclude `-s` if `-v` is added to any command?  Adding `#` might be helpful too, although this is a tiny patch so it probably wouldn't make a difference.  Just a thought for future packages.